zebrad/src/application.rs

```diff
@@ -89,7 +89,11 @@ impl Application for ZebradApp {
         let terminal = Terminal::new(self.term_colors(command));
         // This MUST happen after `Terminal::new` to ensure our preferred panic
         // handler is the last one installed
-        color_eyre::install().unwrap();
+        color_eyre::config::HookBuilder::default()
+            .issue_url("https://github.com/ZcashFoundation/zebra/issues/new")
+            .add_issue_metadata("version", "3.0.0-alpha.0")
```

Comment:
drive-by note: you can get this automatically with 
```rust
env!("CARGO_PKG_VERSION")
```
instead of a hard-coded string

Reply:
blessed, ty, i tried and failed to find how to look this up yesterday.

Reply:
nifty: https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates

Reply:
`CARGO_PKG_REPOSITORY` might be useful too

Reply:
oh, yeah, you could do the `issue_url` with
```rust
concat!(env!("CARGO_PKG_REPOSITORY"), "/issues/new")
``` 
or something!

Reply:
this works and is extremely sick